### PR TITLE
added empty plugin

### PR DESCRIPTION
**Pull request by acselp**

### CODE REVIEW

File: src/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "src",
```
Comment (andrianboscanean) [inline src/package-lock.json:2]: hm, not needed

File: src/Presentation/Nop.Web/Controllers/BlogController.cs [hunk 2]
```diff
@@ -236,7 +236,9 @@ public virtual async Task<IActionResult> BlogCommentAdd(int blogPostId, BlogPost
             //If we got this far, something failed, redisplay form
             RouteData.Values["action"] = "BlogPost";
             await _blogModelFactory.PrepareBlogPostModelAsync(model, blogPost, true);
+            // ReSharper disable Mvc.ViewNotResolved
```
Comment (andrianboscanean) [inline src/Presentation/Nop.Web/Controllers/BlogController.cs:239]: you have edited too much, remove these changes

File: src/Presentation/Nop.Web/Areas/Admin/Controllers/ExternalAuthenticationController.cs [hunk 3]
```diff
@@ -54,6 +54,7 @@ public virtual async Task<IActionResult> Methods()
             var model = _externalAuthenticationMethodModelFactory
                 .PrepareExternalAuthenticationMethodSearchModel(new ExternalAuthenticationMethodSearchModel());
 
+            // ReSharper disable once Mvc.ViewNotResolved
```
Comment (andrianboscanean) [inline src/Presentation/Nop.Web/Areas/Admin/Controllers/ExternalAuthenticationController.cs:57]: same

File: src/Plugins/Nop.Plugin.Products.ProductProvider/ProductProvider.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+{
+    public string GetConfigurationPageUrl()
+    {
+        throw new NotImplementedException();
+    }
+
+    public PluginDescriptor PluginDescriptor { get; set; }
+    public Task InstallAsync()
+    {
+        throw new NotImplementedException();
```
Comment (andrianboscanean) [inline src/Plugins/Nop.Plugin.Products.ProductProvider/ProductProvider.cs:17]: oups, return Task.Completed

**Review by andrianboscanean — COMMENTED** [review]
inca un pic:)

File: src/Plugins/Nop.Plugin.Products.ProductProvider/ProductProvider.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Threading.Tasks;
+using Nop.Services.Plugins;
+
+namespace Nop.Plugin.ProductProvider;
+
+public class ProductProvider : IPlugin
```
Comment (andrianboscanean) [inline src/Plugins/Nop.Plugin.Products.ProductProvider/ProductProvider.cs:7]:
```suggestion
public class ProductProvider : BasePlugin, IPlugin
```

**Comment by RomanovM** [comment]
@acselp @andrianboscanean Why are you making these changes in the main repository and not in your forks?

**Comment by andrianboscanean** [comment]
damn, our fault

sorry

**Pull request closed without merging**